website/pages/docs/billing-and-usage/spending-limits.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Billing and Usage | Spending Limits
+description: CloudQuery is an open source ELT framework that can be used as an alternative to Airbyte. CloudQuery is built for performance, and is easy to deploy and maintain.
```

Comment:
We may want to update this description to better reflect the contents of this page

Reply:
updated